changelog/unreleased/issue-3414

```diff
@@ -1,19 +1,19 @@
-Enhancement: Add --keep-within-hourly switch to restic forget
+Enhancement: Add `--keep-within-hourly` option to restic forget
 
-restic forget allowed users to specify keeping a given number of 
+Restic forget allowed users to specify keeping a given number of
 hourly backups, or to keep all backups within a given interval,
 but not both, that is to specify keeping hourly backups within
 a given interval.
 
-The --keep-within-hourly switch offers this functionality, and 
-parallell switches for daily/weekly/monthly/yearly are also 
-implemneted. The new switches are:
+The `--keep-within-hourly` option offers this functionality and
+similar options for daily/weekly/monthly/yearly are also
+implemented. The new flags are:
```

Comment:
Suggest "The new `--keep-within-hourly` option now offer this functionality. Similar options for daily/weekly/monthly/yearly are also implemented - the new options are:"